ktor-network/ktor-network-tls/jvm/src/io/ktor/network/tls/TLSConfigBuilder.kt

```diff
@@ -79,16 +79,17 @@ public fun TLSConfigBuilder.addCertificateChain(chain: Array<X509Certificate>, k
 }
 
 /**
- * Add client certificates from [store].
+ * Add client certificates from [store] by using the certificate with specific [alias]
+ * or all certificates, if [alias] is null.
  */
-public fun TLSConfigBuilder.addKeyStore(store: KeyStore, password: CharArray) {
+public fun TLSConfigBuilder.addKeyStore(store: KeyStore, password: CharArray, alias: String? = null) {
```

Comment:
Please split this method in two with the new parameter and without it. It's required to keep the binary compatibility.

Reply:
Splitted

Reply:
Currently this new function is addictive only. Do you want to deprecate and hide the "old" function too? Then the alias must have `null` by default, to ensure no source changes in usage

Reply:
Both ways are ok for me. What matters is to not remove anything from public API (both source and binary) in non-major releases.